Library/Homebrew/tab.rb

```diff
@@ -341,6 +341,20 @@ def write
     tabfile.atomic_write(to_json)
   end
 
+  # Helper method for checking truthiness of a value that may not be specified.
+  # `Tab.might_be?(:foo)` differs from `tab.foo` in that if `foo` is not
+  # specified `#might_be?` will return true.
+  def might_be?(key)
```

Comment:
Seems fine or `nil_or_truthy?`. Ideally I think `key` would be in the name but don't feel incredibly strongly.